### PR TITLE
[dep] systemd

### PR DESCRIPTION
**Pull request by 130s**
## Package name:

`systemd`

## Purpose of using this:

- The pkg contains `systemctl`, which is operational interface for OS daemon on some Linux distros.
   - There already seems to be a rosdep key `libsystemd-dev`, which doesn't seem to include `systemctl`.
- Usecase e.g. My team is making a tool that controls daemon. This pkg doesn't seem to come with Docker image of Ubuntu so the dependency on it needs to be defined.

# Distro packaging links:

- Debian: https://packages.debian.org/
  - REQUIRED
  - https://packages.debian.org/stretch/systemd
- Ubuntu: https://packages.ubuntu.com/
   - REQUIRED
   - https://packages.ubuntu.com/search?keywords=systemd
- Fedora: https://src.fedoraproject.org/browse/projects/
  - IF AVAILABLE
  - https://src.fedoraproject.org/rpms/systemd#bodhi_updates
- Arch: https://www.archlinux.org/packages/
  - IF AVAILABLE
  - https://archlinux.org/packages/core/x86_64/systemd/
- Gentoo: https://packages.gentoo.org/
  - IF AVAILABLE
  - https://packages.gentoo.org/packages/sys-apps/systemd
- macOS: https://formulae.brew.sh/
  - IF AVAILABLE
  - https://formulae.brew.sh/formula-linux/systemd
- RHEL http://mirror.centos.org/centos-8/8/BaseOS/x86_64/os/Packages/systemd-239-45.el8.i686.rpm
- OpenSuse https://software.opensuse.org/package/systemd

### CODE REVIEW

**Comment by audrow** [comment]
Hey @130s, can you update the PR so that the Github Actions are happy?

**Review by ivanpauno — COMMENTED** [review]
@130s can you follow the three automatic suggestions?

- Key 'systemd' might be satisifed by rhel package named 'systemd': http://mirror.centos.org/centos-8/8/BaseOS/x86_64/os/Packages/systemd-239-45.el8.i686.rpm
- Key 'systemd' might be satisifed by opensuse package named 'systemd': https://software.opensuse.org/package/systemd
- Key 'systemd' might be satisifed by fedora package named 'systemd': https://src.fedoraproject.org/rpms/systemd#bodhi_updates

File: rosdep/base.yaml [hunk 1]
```diff
@@ -7333,6 +7333,15 @@ sysstat:
   nixos: [sysstat]
   openembedded: [sysstat@openembedded-core]
   ubuntu: [sysstat]
+systemd:
+  arch: [systemd]
+  debian: [systemd]
+  fedora: [systemd]
+  gentoo: [sys-apps/systemd]
+  macports: [systemd]
```
Comment (ivanpauno) [inline rosdep/base.yaml:7341]: macports is not the same than homebrew

```suggestion
  osx:
    homebrew:
      packages: [systemd]
```

**Review by ivanpauno — APPROVED** [review]
lgtm with comment addressed

**Review by ivanpauno — APPROVED** [review]

**Review by sloretz — APPROVED** [review]

**Pull request merged**